### PR TITLE
Booking view

### PR DESCRIPTION
**Pull request by ConnorWalsh7**
-Designed booking view to closly match tutor profile view
-Displays a tutors rating, school, rate, location, courses, and availability
-Book now button takes you to the StudentPaymentFragment
-Passing any variables needed to create a new session through bundles into this view
-On a successful payment a new session is created and the user is redirected to the session details
-Added a DateFormatHelper for converting our dates into more readable strings
-Added various helper methods in some of the models for getting the data I needed for the booking view
-Added a seeder for available_times to test the booking view

### CODE REVIEW

File: app/src/main/java/cs/dal/krush/helpers/DateFormatHelper.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.Locale;
+
+/**
+ * This class contains functions for converting dates into more readable formats
+ */
+public class DateFormatHelper
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/helpers/DateFormatHelper.java]: awesome Connor, thanks a lot for creating this class!

File: app/src/main/java/cs/dal/krush/studentFragments/StudentPaymentFragment.java [hunk 2]
```diff
@@ -71,8 +77,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         paymentMethodsLabel.setTypeface(typeFace);
 
         //set cost label
-        // TODO: 2017-03-17 Get tutoring session cost from bundle 
-        tutoringCost.setText("$" + 99.99);
+        tutoringCost.setText("$" + COST + "0");
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/studentFragments/StudentPaymentFragment.java]: is there a reason we have to append "0" to the cost?

Comment (ConnorWalsh7) [inline app/src/main/java/cs/dal/krush/studentFragments/StudentPaymentFragment.java]: Not anymore. Now using a currency formatter to directly convert the float price into a string to display the cost that nicely formats to 2 decimal places and auto includes the currency symbol depending on locale, so it should work with pounds or euros or other formats

**Comment by Mons1Oerjan** [comment]
Approved!

**Pull request merged**